content/docs/user-guide/networking/settings.md

```diff
@@ -5,9 +5,41 @@ linktitle: Network Settings
 ---
 
 ## Overview
-This page documents [console variables](/docs/user-guide/appendix/cvars) and settings for AzNetworking and Multiplayer behavior.
+## Overview
+This page documents [console variables](/docs/user-guide/appendix/cvars) and other settings that control Networking and Multiplayer behavior.
+
+## Networking commands
+These are special console commands that control the flow of networked and multiplayer games. 
+
+| Setting                | Description                                                                                | Default | Notes |
+|------------------------|--------------------------------------------------------------------------------------------|---------|-------|
+| host                   | Opens a multiplayer connection as a host for other clients to connect to.                  |||                                                                      ||
+| connect                | Opens a multiplayer connection to a host                                                   |||
```

Comment:
This command also optionally takes a hostname/IP address and port number to connect to if you don't want to just connect to localhost on the default port.

Reply:
Added